__tests__/app.test.js

```diff
@@ -57,12 +57,10 @@ describe("GET /api/articles/:article_id", () => {
       .get("/api/articles/3")
       .expect(200)
       .then(({ body: { article } }) => {
-        expect(article).toEqual({
-          article_id: 3,
+        expect(article).toMatchObject({
```

Comment:
Hopefully this illustrates why it's best to use more flexible matchers like `toMatchObject` from the beginning, as ideally we shouldn't have to alter previous tests when we add new functionality 🔍